recipes/find_circ/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "find_circ" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marvin-jens/find_circ.git
```

Comment:
I can see a couple of releases on this repo- is there a reason [a stable URL](https://bioconda.github.io/contributor/guidelines.html#stable-urls) can't be used?